packages/ui/src/layout/banners/Announcement.Countdown.tsx

```diff
@@ -27,6 +27,7 @@ function CountdownBanner() {
   const { pathname } = useRouter()
   const isLaunchWeekPage = pathname === '/launch-week'
   const isLaunchWeekSection = pathname.includes('launch-week')
+  const basePath = typeof window !== 'undefined' ? window.location.origin : ''
```

Comment:
ah wait..

Reply:
should we not just have a basePath prop for this

Reply:
i think its fine though - since if we have this as a prop, it'll be passed down the same way
the parent component will have to do `window.location.origin` - the implementation won't differ

Reply:
it would use next router though @joshenlim